src/OrchardCore/OrchardCore.Testing/Apis/SiteContextOptions.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections.Concurrent;
+using OrchardCore.Testing.Apis.Security;
+
+namespace OrchardCore.Testing.Apis;
+
+public class SiteContextOptions
+{
+    static SiteContextOptions()
+    {
+        PermissionsContexts = new();
+    }
+
+    public static ConcurrentDictionary<string, PermissionsContext> PermissionsContexts { get; set; }
+
```

Comment:
add a static action to be able to configure the IHostBuilder
public static Action<IHostBuilder> ConfigureHostBuilderAction { get; set; }
So in a Context, in a static constructor one could configure the host eg. to use NServicebus

Reply:
We might defer this for now and add it later, the aim is to extract what we have nowadays into `OC.Testing` then we can add helper features